cnf-certification-test/accesscontrol/suite.go

```diff
@@ -235,3 +241,19 @@ func TestPodHostPID(env *provider.TestEnvironment) {
 	tnf.ClaimFilePrintf("bad pods: %v", badPods)
 	gomega.Expect(badPods).To(gomega.BeNil())
 }
+
+func TestPodServiceAccount(env *provider.TestEnvironment) {
+	ginkgo.By("Tests that each pod utilizes a valid service account")
+	failedPods := []string{}
+	for _, put := range env.Pods {
+		ginkgo.By(fmt.Sprintf("Testing service account for pod %s (ns: %s)", put.Name, put.Namespace))
+		if put.Spec.ServiceAccountName == "" {
```

Comment:
I know this is the same code as test-network-function, but I think this will be always 'default' or some other name